docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -138,17 +138,17 @@ BAU parameter data tables
    :widths: 30, 20, 10, 10
    :header-rows: 1
 
-.. csv-table:: Probability of testing LDL-c 
-   :file: prob_testing_ldlc.csv
-   :widths: 20, 10, 10
+.. csv-table:: Table 4: Probability of Adherence (Table 4a, 4b) parameters
+   :file: adherence_parameters.csv
+   :widths: 30, 20, 10, 10
    :header-rows: 1
 
-.. csv-table:: Adherence parameters
-   :file: adherence_parameters.csv
+.. csv-table:: Table 6: Current treatment practice - distribution by drug type 
+   :file: current_rx.csv
    :widths: 30, 20, 10, 10
```

Comment:
The specific Rx for each patient (at initialization and for new patients during the simulation) is determined by the data in Table 6 - current treatment practice distribution by drug type.  First, the type of drug is determined (statin, ezetimibe or fibrate).  Then the sub-type of statin is determined for patients on statin.  In BAU, dosing is 40mg for low potency statin (called "high dose") and 20mg for high potency statin (called "low dose").  In the 2 intervention scenarios, the initial dose is "high dose" of high potency statin.